### ISSUE TITLE
Will you add support for EXT_buffer_storage?

### ISSUE THREAD

**Issue by Abz1** [hrydgard/ppsspp#8819]
i heard that this extension, EXT_buffer_storage will help Adreno devices. Will you add support for it? and in general add support for OpenGL ES 3.2?

**Comment by hrydgard** [comment]
Oh, it's available on Adreno now? Yes, it can probably help performance a bit, though it needs a pretty big redesign of the OpenGL rendering backend of PPSSPP. Unfortunately I won't be able to get this done in the near future, but it will happen.

**Comment by Abz1** [comment]
Yeah, its been available for awhile now..i was talking to Ryan and he said buffer_storage will help with Adreno devices. could you also support GLES 3.2?

**Comment by unknownbrackets** [comment]
How often is this available where Vulkan isn't?  Wondering how much motivation we have to implement it.

-[Unknown]

**Comment by hrydgard** [comment]
When the refactoring in the gl-render-manager branch is done, making good use of this extension will be very easy, at least for vertex data. We'd also want to use it for UBOs but the GL code still uses regular "loose" uniforms on all platforms.

**Comment by Gamemulatorer** [comment]
> Oh, it's available on Adreno now? Yes, it can probably help performance a bit, though it needs a pretty big redesign of the OpenGL rendering backend of PPSSPP to fully get the benefit. Unfortunately I won't be able to get this done in the near future, but it will happen.
> 
> hm, yeah:
> 
> http://opengles.gpuinfo.org/gles_listreports.php?extension=GL_EXT_buffer_storage

This is included in https://github.com/hrydgard/ppsspp/pull/16156 on android? or already included since v1.7?
